testbed/addons/awslb/construct.ts

```diff
@@ -0,0 +1,47 @@
+import cdk = require('@aws-cdk/core');
+import eks = require('@aws-cdk/aws-eks');
+import iam = require('@aws-cdk/aws-iam');
+import * as yaml from 'js-yaml';
+import * as request from 'sync-request';
+
+export interface AWSLoadBalancerControllerProps {
+    cluster: eks.Cluster;
+    namespace: string;
+}
+
+export class AWSLoadBalancerController extends cdk.Construct {
+    constructor(scope: cdk.Construct, id: string, props: AWSLoadBalancerControllerProps) {
+        super(scope, id);
+
+        const sa = props.cluster.addServiceAccount('aws-lbc', {
+            name: "aws-lbc",
+            namespace: props.namespace
+        });
+
+        sa.role.addManagedPolicy({
+            //to-do  move this to managed policy once it's added to IAM :https://github.com/awslabs/kubernetes-iteration-toolkit/issues/6
```

Comment:
nit, typical OSS convention is TODO(reason/name/issue), many editors will highly this.

e.g. // TODO(#6) Use a managed policy.

Reply:
From what I know from upstream k8s is - `TODO: reason/issue`.

Reply:
Perfect. Do this :)